### PR TITLE
added examples for New-AzSqlVM cmdlet

### PR DESCRIPTION
**Pull request by veppala**
## Checklist

- [X] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** select appropriate branch. Cmdlets from Autorest.PowerShell should go to [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch. 
* **SHOULD** make the title of PR clear and informative, and in the present imperative tense. 
* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. Add changelog in description section if PR goes into [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** have approved design review for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) ([_Microsoft internal only_](../blob/main/CONTRIBUTING.md#onboarding)) with following situations
    * Create new module from scratch
    * Create new resource types which are not easy to conform to [Azure PowerShell Design Guidelines](../blob/main/documentation/development-docs/design-guidelines)
    * Create new resource type which name doesn't use module name as prefix
    * Have design question before implementation
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** introduce [breaking changes](../blob/main/documentation/breaking-changes/breaking-changes-definition.md) in Az minor release except preview version.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

File: src/SqlVirtualMachine/examples/New-AzSqlVM.md [hunk 1]
```diff
@@ -20,3 +21,146 @@ Location	Name		ResourceGroupName
...
+New-AzSqlVM -ResourceGroupName 'ResourceGroup01' -Name 'sqlvm1' -Location 'eastus' -AdditionalFeatureServerConfigurationIsRServicesEnabled
+```
+
+```output
+Location	Name		ResourceGroupName
+--------	----		-----------------
+eastus		sqlvm1		ResourceGroup01	
+```
+
+Create a SQL virtual machine with Enable R services in SQL2016 onwards.
```
Comment (ranishan) [inline src/SqlVirtualMachine/examples/New-AzSqlVM.md:63]: Better to check with Amitabh/PM. We have stopped exposing this in portal so probably no needed in the example

File: src/SqlVirtualMachine/examples/New-AzSqlVM.md [hunk 2]
```diff
@@ -20,3 +21,146 @@ Location	Name		ResourceGroupName
...
+New-AzSqlVM -ResourceGroupName 'ResourceGroup01' -Name 'sqlvm1' -Location 'eastus' -SqlManagementType 'Lightweight'
+```
+
+```output
+Location	Name		ResourceGroupName
+--------	----		-----------------
+eastus		sqlvm1		ResourceGroup01	
+```
+
+Create a SQL virtual machine with Lightweight mode.
```
Comment (ranishan) [inline src/SqlVirtualMachine/examples/New-AzSqlVM.md:153]: We are no longer exposing SQLManagementType so we should remove this example. Please check with Amitabh/PM

File: src/SqlVirtualMachine/examples/New-AzSqlVM.md [hunk 3]
```diff
@@ -20,3 +21,146 @@ Location	Name		ResourceGroupName
...
+Location	Name		ResourceGroupName
+--------	----		-----------------
+eastus		sqlvm1		ResourceGroup01	
+```
+
+Create a SQL virtual machine and configure auto patching settings.
+
+### Example 8
+```powershell
+$password = "P@ssw0rd"
```
Comment (ranishan) [inline src/SqlVirtualMachine/examples/New-AzSqlVM.md:98]: This was also removed from Portal experience. Better to check with PM team/ Amitabh if we want this in the examples.

File: src/SqlVirtualMachine/examples/New-AzSqlVM.md [hunk 4]
```diff
@@ -20,3 +21,146 @@ Location	Name		ResourceGroupName
...
+--------	----		-----------------
+eastus		sqlvm1		ResourceGroup01	
+```
+
+Create a SQL virtual machine and configure assessment settings.
+
+### Example 10
+```powershell
+New-AzSqlVM -ResourceGroupName 'ResourceGroup01' -Name 'sqlvm1' -Location 'eastus' -KeyVaultCredentialSettingAzureKeyVaultUrl 'keyvault url' `
+-KeyVaultCredentialSettingCredentialName 'cred1' `
```
Comment (ranishan) [inline src/SqlVirtualMachine/examples/New-AzSqlVM.md:128]: I do not find this in Portal as well.

File: src/SqlVirtualMachine/examples/New-AzSqlVM.md (hunk 1 above)
Comment (veppala) [reply src/SqlVirtualMachine/examples/New-AzSqlVM.md]: removed deprecated features from examples.

**Review by Nickcandy — APPROVED** [review]

**Pull request merged**